ddl/backfilling.go

```diff
@@ -444,17 +444,27 @@ func tryDecodeToHandleString(key kv.Key) string {
 }
 
 // sendRangeTaskToWorkers sends tasks to workers, and returns remaining kvRanges that is not handled.
-func (w *worker) sendRangeTaskToWorkers(workers []*backfillWorker, reorgInfo *reorgInfo,
+func (w *worker) sendRangeTaskToWorkers(t table.Table, workers []*backfillWorker, reorgInfo *reorgInfo,
 	totalAddedCount *int64, kvRanges []kv.KeyRange) ([]kv.KeyRange, error) {
 	batchTasks := make([]*reorgBackfillTask, 0, len(workers))
 	physicalTableID := reorgInfo.PhysicalTableID
 
 	// Build reorg tasks.
 	for _, keyRange := range kvRanges {
+		endKey := keyRange.EndKey
+		endK, err := getRangeEndKey(workers[0].sessCtx.GetStore(), workers[0].priority, t, keyRange.StartKey, endKey)
+		if err != nil {
+			logutil.BgLogger().Info("[ddl] send range task to workers, get reverse key failed", zap.Error(err))
+		} else {
+			logutil.BgLogger().Info("[ddl] send range task to workers, change end key",
```

Comment:
Current log is a little bit hard to read:

```
[2021/08/05 19:31:11.821 +08:00] [INFO] [backfilling.go:459] ["[ddl] send range task to workers, change end key"] ["end key"="t\ufffd\u0000\u0000\u0000\u0000\u0000\u00008_r\ufffd\u0000\u0000\u0000\u0000\u0000\u0000\u0000"] ["current end key"="t\ufffd\u0000\u0000\u0000\u0000\u0000\u00008_r\ufffd\u0000\u0000\u0000\u0000\u0005\ufffd\ufffd"]
```

How about use `tryDecodeToHandleString`, the log will be:

```log
[2021/08/05 19:34:49.797 +08:00] [INFO] [backfilling.go:459] ["[ddl] send range task to workers, change end key"] ["end key"=5188146770730811392] ["current end key"=4611686018427754381]

```